exporter/util/attestation/extract.go

```diff
@@ -0,0 +1,260 @@
+package attestation
```

Comment:
Packages that import solver, ImmutableRef etc. should not be under util. They can still be in a separate subpackage if it makes sense.